### ISSUE TITLE
undesired scroll to top when selecting track in playlist

### ISSUE THREAD

**Issue by tropicaldev** [scottschiller/SoundManager2#123]
The Undesired scroll to top happens when the track in the playlist has the download icon and the track is at the bottom of the playlist

**Comment by tropicaldev** [comment]
as s  workaround i removed the div '<div class="sm2-row">' after the element <li> that renders the track in the playlist

the playlist appears to be working ok ... will i have any problems by removing this div?

**Comment by scottschiller** [comment]
Sounds like a bug, thanks for the note. I don't think removing the `.row` `<div>` will kill the player, but it might mess with the layout a little bit.

**Comment by samayo** [comment]
@scottschiller  Maybe you are busy and all, but could you consider delegating responsibility of this repo to someone else? This is a very nice repo which a lot less effort to be the perfect tool for everybody. Don't kill it :)

**Comment by dschiller** [comment]
Simply comment out Line 550 in bar-ui.js:

// scrollTop = dom.playlist.scrollTop; 

It's the Line in the Function " function select(item) { " in Line 528.

Cheers
Dirk

**Comment by samayo** [comment]
@dschiller Do you really think OP is still stuck with the same issue after 4 months?

**Comment by dschiller** [comment]
Sure not. But I did. And maybe some other People. But thanks for your Advice.

**Comment by samayo** [comment]
Yeah you did the right thing, I'm just a little pissed at how slow this greate library is being maintained, if we still can call it that. Thanks for your help though

**Comment by dschiller** [comment]
Yeah maybe Scott is very Busy. And I think we could be very thankfull for his great Work.

**Comment by scottschiller** [comment]
Indeed; apologies for super-late reply, have been very busy with professional work. 😅
(And hey, another Schiller! Small world, etc.)

The SoundManager project began in 2001, I created version 2 in 2006, I think - and it's been quite a wild ride, seeing it get used in all sorts of interesting places online. These days it is quite stable, and it's been a year so far without a major update ( 😱 ) - usually I get one or two out a year, at least. 😆 

These days, mostly it's updates to demos - new demos, or minor bug fixes, tweaks for mobile/iOS and the like.

**Comment by samayo** [comment]
Well, good to see you back. This is a great library and needs someone with time to look after it. There reason you may not be getting many bug reports is because people saw the activities, lack of responses and thought it was becoming abandoned. Anyway, I hope that wont be true.

**Comment by scottschiller** [comment]
The cartoon avatar notwithstanding, I perceive your comment as negative and snarky and feel it could be improved. Also, it is incorrect. Issue reports are pretty minor, largely as the project is quite stable given its age.

The "Bar UI" is a more recent demo, so it's entirely likely there will be some bugs in that one; that's totally fair.

That said, my $.02 is that bemoaning lack of support on a FOSS project in generally unproductive.

In case it's unclear: This has been an unpaid, personal-time, volunteer-based project for the last decade. I've been busy with other things as of late, and the project is pretty stable. The issue reported on this demo is pretty small and has been addressed, there are probably larger fish to fry for the next update.

_However_, this one's open because I haven't fixed and pushed an official update for it yet. So, the system works!

**Comment by dschiller** [comment]
Hey Scott - same-namer ;) I totally understand you. I also understand you Samayo. I took a look to your GitHub Projects and they are also not well maintained and I recognized that you are a Programmer. Maybe  you could support Soundmanager and also Scott instead of scatter neg. Energies ;) Don't think that someone will fix all Bugs in a Minute without getting Money - that's essentiell, if someone jump on his Head. I totally understand that you are frustrated but the Soundmanger is very robust and if you understand the Code you can easily fix the Bugs by your own or create some Workarounds. If you need Help let me know.

Cheers
Dirk

**Comment by samayo** [comment]
Thanks. @dschiller  ... eh, I was just trying to help.
